common/shlibs

```diff
@@ -3395,3 +3395,8 @@ libgaminggearwidget.so.0 libgaminggear-0.15.1_1
 libopkg.so.1 libopkg-0.4.0_1
 libQMatrixClient.so.0.4 libqmatrixclient-0.4.2.1_1
 libipset.so.13 libipset-7.1_1
+libVkLayer_unique_objects.so Vulkan-ValidationLayers-1.1.97_1
```

Comment:
please do not them imediatelly, they are not versioned

Reply:
Sorry I do not understand what you are meaning. I do not need to add anything to shlibs because the libs have no version (*.so.x)?